Dockerfile

```diff
@@ -70,18 +70,8 @@ COPY . /code/listenbrainz
 ###########################################
 FROM listenbrainz-base as listenbrainz-prod
 
-
-# production sidenote: We create a `lbdumps` user to create data dumps because
-# the ListenBrainz servers on prod (etc. lemmy) use storage boxes [0] which
-# are owned by lbdumps on the host too.
-# [0]: https://github.com/metabrainz/syswiki/blob/master/ListenBrainzStorageBox.md
-RUN groupadd --gid 900 lbdumps
-RUN useradd --create-home --shell /bin/bash --uid 900 --gid 900 lbdumps
-RUN mkdir /logs && chown lbdumps:lbdumps /logs
-
-# Create directories for backups and FTP syncs
-RUN mkdir /home/lbdumps/backup /home/lbdumps/ftp
-RUN chown -R lbdumps:lbdumps /home/lbdumps/backup /home/lbdumps/ftp
+# Create directories for cron logs, dump backups and FTP syncs
+RUN mkdir /logs /mnt/backup /mnt/ftp
```

Comment:
I want to look into this in more detail - from what I understand, we don't need to make a directory if we want to mount a volume into it, however there are a few exceptions:

* if we define a directory as a volume with the docker `VOLUME` command, then docker will automatically create  an anonymous volume at runtime and copy the contents of this directory in the image to this volume the first time it's used. This is useful to do things like set permissions and add initial data (e.g. imagine that we want to add subdirectories for the incremental/full/spark folders automatically)
* If we attach a volume to this location when we `docker run`, it'll use this named volume as a location. Because we use the shared docker image, if we declare these locations with `VOLUME` then docker will start 3 new anonymous volumes for _every_ container that we start (web, timescale writer, spotify reader, etc). This seems a bit silly to me as well. We could prevent it from doing so by mounting the named volumes into all of these containers, but that also seems like the wrong decision.
* Keep in mind that if we don't `mkdir` here, and if we don't declare a location with `VOLUME` and we forget to mount a volume or if we accidentally run a dump command in another container then the dump will happen inside the context of the container and the files will disappear if the container is shut down. I think this is no worse than the situation we had before this PR

I think the best solution here may be to remove these specific mkdirs if we can just mount a volume into the location. I'll test it.